### ISSUE TITLE
Can not connect to unix socket file

### ISSUE THREAD

**Issue by herb123456** [hipages/php-fpm_exporter#19]
```
 # ./php-fpm_exporter_0.4.1_linux_amd64 server --web.listen-address localhost:9101 --phpfpm.scrape-uri unix:///var/run/php71-fpm.sock
INFO[0000] Starting server on localhost:9101 with path /metrics
```

then test using curl output

```
ERRO[0002] dial unix :: connect: no such file or directory
ERRO[0002] Error scraping PHP-FPM: dial unix :: connect: no such file or directory
```
just because uri.Hostname() and uri.Port() is empty when address is unix socket file path at [phpfpm/phpfpm.go Line160](https://github.com/hipages/php-fpm_exporter/blob/master/phpfpm/phpfpm.go#L160) 

I will send a pull request for this issue later.

**Comment by estahn** [comment]
@herb123456 Unix sockets were not a priority for us, so thanks a lot for your pull request.

Looking at the code we probably need to find a way to integrate #21 

Thoughts?

**Comment by herb123456** [comment]
that's ok. I will send another pull request a few days later.

**Comment by estahn** [comment]
@herb123456 This issue has been fixed with **v0.5.0**. Thanks again for your pull request.
